### PR TITLE
chore(linux): Refactor deb-packaging.sh script 🎬

### PR DESCRIPTION
**Pull request by ermshiperete**
Make use of builder_run_action.

@keymanapp-test-bot skip

### CODE REVIEW

**Review by darcywong00 — APPROVED** [review]
lgtm

File: linux/scripts/deb-packaging.sh [hunk 1]
```diff
@@ -50,25 +48,23 @@ if builder_start_action source; then
...
     echo ":heavy_check_mark: ${LIB_NAME} API didn't change" >&2
   fi
-  builder_finish_action success verify
-  exit 0
-fi
+}
+
+builder_run_action dependencies  dependencies_action
+builder_run_action source        source_action
+builder_run_action verify        verify_action
```
Comment (mcdurdin) [inline linux/scripts/deb-packaging.sh:70]: I notice you are no longer doing `exit 0` after each action. Is this change in behaviour intentional?

Comment (ermshiperete) [reply linux/scripts/deb-packaging.sh:70]: Yes, this brings it more in line with how the other `build.sh` scripts work. I can't see a reason why you shouldn't be able to run multiple actions.

Comment (mcdurdin) [reply linux/scripts/deb-packaging.sh:70]: Sounds sensible to me. Just wasn't sure if there was a particular reason it was doing that before :grin:

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.205-alpha](https://keyman.com/downloads/releases/alpha/17.0.205)

**Pull request merged**